pyface/data_view/exporters/row_exporter.py

```diff
@@ -0,0 +1,50 @@
+from traits.api import Bool
+
+from pyface.data_view.abstract_data_exporter import AbstractDataExporter
+
+
+class RowExporter(AbstractDataExporter):
+
```

Comment:
can you add a class docstring, like with the `ItemExporter`?

Reply:
you can move a significant part of the `get_data` docstring here, if you didn't want to write a new docstring.